### ISSUE TITLE
Since 0.3.0, refreshToken( err, result ) gives `global` back as result

### ISSUE THREAD

**Issue by WeeJeWel** [lelylan/simple-oauth2#68]
A pretty odd issue that I've traced back to 0.3.0. In 0.2.1, it's working.

Reproduce:

```
// Get the access token object.
// Initialize the OAuth2 Library
var oauth2 = require('simple-oauth2')(credentials);
var token;
var tokenConfig = {};

// Get the access token object for the client
oauth2.client.getToken(tokenConfig, function saveToken(err, result) {
	if (err) console.error(err);
	token = oauth2.accessToken.create(result);
	console.log('got token', token)
	token.refresh(function(err, result) {
		// result here is the same as `global`, with an added `token` property, which contains the new token
		console.log('refreshed token', err, result)
	})
});
```

**Comment by jonathansamines** [comment]
HI @WeeJeWel not sure about what this issue is about. What do you mean by `global`? The node `global` object?

**Comment by roymap** [comment]
I see that too.  The result contains a token object instead of being that token object.

**Comment by WeeJeWel** [comment]
@jonathansamines `result` == `global` (node.js global variable), and `global.token` is the token i would expect as `result`

**Comment by jonathansamines** [comment]
I see, let me check this issue.

**Comment by jonathansamines** [comment]
This issue will be solved once #72 is merged to develop.

**Comment by jonathansamines** [comment]
Closed by #72
